### ISSUE TITLE
Error when building examples

### ISSUE THREAD

**Issue by JohanThestrup** [sidiousvic/phantom#86]
When attempting to run the examples provided in the repo the following error message appears in my terminal. The error message is consistent in all examples (todo, pizza, calculator) on my computer. 

The error seems to be related to webpack and I'd be happy to take on the issue with some assistance (webpack newbie).

Or it just might be that I have misinterpreted the process on how to run the examples - in which case I could create a suggestion for a small change in the readme!

Recreate the error:

```
clone the repo
npm i
npm run build
npm run example/[example name]
```

```
> @sidiousvic/phantom@2.1.5 example/calculator
> webpack --config examples/calculator/webpack.config.js && webpack-dev-server --mode development --hot --watch-stdin --config examples/calculator/webpack.config.js

Hash: 7911f0591e626159bf61
Version: webpack 4.43.0
Time: 307ms
Built at: 05/28/2021 10:28:49 AM
                               Asset       Size  Chunks             Chunk Names
                        ./index.html  253 bytes          [emitted]  
6c6b90dd69642b4893656e14f031893f.ttf   61.1 KiB          [emitted]  
                      main.bundle.js   34.5 KiB    main  [emitted]  main
Entrypoint main = main.bundle.js
[./examples/calculator/die_nasty.ttf] 80 bytes {main} [built]
[./examples/calculator/functions.js] 2.18 KiB {main} [built]
[./examples/calculator/index.js] 2.07 KiB {main} [built]
[./examples/calculator/phantomStore.js] 1.93 KiB {main} [built]
[./examples/calculator/styles.css] 526 bytes {main} [built]
[./lib/phantom.js] 467 bytes {main} [built] [failed] [1 error]
[./node_modules/css-loader/dist/cjs.js!./examples/calculator/styles.css] 4.65 KiB {main} [built]
    + 3 hidden modules

ERROR in ./lib/phantom.js 234:63
Module parse failed: Unexpected token (234:63)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| 
|   for (let nodeAttributes = 0; nodeAttributes < node.attributes.length; nodeAttributes++) {
>     const attributeName = node.attributes.item(nodeAttributes)?.name;
| 
|     if (allowedTags[nodeName].hasOwnProperty(attributeName)) {
 @ ./examples/calculator/index.js 1:0-40 12:4-11
```

PS.
I'm brand new to OSCB and open to any feedback regarding how I can improve the way I'm submitting issues.
Thank you!

**Comment by mussinbenarbia** [comment]
@sidiousvic  Is it ok if I work on this issue?

**Comment by sidiousvic** [comment]
Hey @JohanThestrup, this is an error that was on the latest dev branch, I JUST updated it. Can you run this locally:

```
git checkout production
git branch -d dev 
git checkout dev
npm run example/[example]
```

And report back?

**Comment by sidiousvic** [comment]
Thank you @mussinbenarbia, if the above steps don't work, please go ahead!

**Comment by mussinbenarbia** [comment]
Just to be sure I deleted the fork, re-forked, re-cloned and was still getting the same issue so I opened a PR to fix it if that's ok! PR -> https://github.com/sidiousvic/phantom/pull/87

**Comment by sidiousvic** [comment]
Closed via #87.
